spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,55 @@
+
+describe "user stories" do
+
+  # PERSONAS
+  # Kylie and Marc, ordering a take out meal
+
+  let(:menu) { Menu.new }
+  let(:order) { Order.new }
+
+  # As a customer
+  # So that I can check if I want to order something
+  # I would like to see a list of dishes with prices
+  it 'so that Marc can see if wants to order something, he should be able to see a list of dishes with prices' do
```

Comment:
Metrics/LineLength: Line is too long. [113/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)